### PR TITLE
doc: warning for unsupported node versions

### LINKED ISSUE #7: randomUUID is not a function
I'm using the local driver, and when I do the `put` method, I get the following error.

Screen Shot 2021-07-02 at 13 18 03

### PR DESCRIPTION
**Pull request by maasencioh**
closes #7

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,30 +4,19 @@
...
-      ]
+      "config": ["datadrive.txt"]
     },
     "types": "adonis-datadrive",
-    "providers": [
-      "adonis-datadrive"
-    ]
+    "providers": ["adonis-datadrive"]
   },
+  "engines": { "node": ">=15.6.0" },
```
Comment (targos) [inline package.json]: the module also works with recent versions of Node.js 14

Comment (maasencioh) [reply package.json]: There's nothing about it on the [docs](https://nodejs.org/api/crypto.html#crypto_crypto_randomuuid_options), from which version?

Comment (targos) [reply package.json]: https://nodejs.org/dist/latest-v14.x/docs/api/crypto.html#crypto_crypto_randomuuid_options

14.17.0

Comment (maasencioh) [reply package.json]: Thank you

File: package.json [hunk 2]
```diff
@@ -4,30 +4,19 @@
...
-      ]
+      "config": ["datadrive.txt"]
     },
     "types": "adonis-datadrive",
-    "providers": [
-      "adonis-datadrive"
-    ]
+    "providers": ["adonis-datadrive"]
   },
+  "engines": { "node": ">=15.6.0||^14.17.0 <15" },
```
Comment (targos) [inline package.json]:
```suggestion
  "engines": { "node": ">=15.6.0||^14.17.0" },
```

**Review by targos — APPROVED** [review]

**Pull request merged**